server/graphql/schema/ingestion/resolvers/mutations/ingestData.ts

```diff
@@ -1359,14 +1364,15 @@ class IngestDataWorker extends ResolverBase {
                     }
                 }
             }
+            if (typeof AVInfo.skipSceneGenerate === 'boolean') IAR.skipSceneGenerate = AVInfo.skipSceneGenerate;
```

Comment:
What's the purpose of the test `if (typeof AVInfo.skipSceneGenerate === 'boolean')` here? Why not just assign `IAR.skipSceneGenerate = AVInfo.skipSceneGenerate;`?

Reply:
My original reasoning was that it's still possible for AVInfo to have null/undefined so this was added to handle those cases.
But thinking about it, it does seem a bit overly specific. I'll make the revision